app/Models/Group.php

```diff
@@ -68,18 +68,19 @@ public function signups()
     }
 
     /**
-     * Returns Customer.io payload attributes for given group, if exists.
+     * Transform this group for Customer.io.
      *
-     * @param Group $group
      * @return array
      */
-    public static function toCustomerIoPayload(?self $group)
+    public function toCustomerIoPayload()
```

Comment:
This change allows this to be used consistently with other `toCustomerIoPayload` methods, which all exist on the instance. To use this in scenarios where the group may not exist, we call it with [`optional`](https://laravel.com/docs/8.x/helpers#method-optional), like so:

```php
optional($this->group)->toCustomerIoPayload()
```